lib/ansible/modules/windows/win_feature.py

```diff
@@ -82,25 +82,27 @@
 '''
 
 EXAMPLES = r'''
-# This installs IIS.
-# The names of features available for install can be run by running the following Powershell Command:
-# PS C:\Users\Administrator> Import-Module ServerManager; Get-WindowsFeature
-$ ansible -i hosts -m win_feature -a "name=Web-Server" all
-$ ansible -i hosts -m win_feature -a "name=Web-Server,Web-Common-Http" all
-ansible -m "win_feature" -a "name=NET-Framework-Core source=C:/Temp/iso/sources/sxs" windows
```

Comment:
Dunno, I must not have been around for that discussion, because I probably would have objected. Pretty much any module *can* work adhoc, but dealing with all the arg transformation and shell escaping mess to do so (especially with Windows paths) probably doesn't belong in the individual module docs IMHO.